### ISSUE TITLE
connect-mongo@4.6.0 not compatible with mongodb@5.0.0

### ISSUE THREAD

**Issue by apydo** [jdesboeufs/connect-mongo#465]
- **I'm submitting a ...**
  [ ] bug report
  [x] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**
Make connect-mongo compatible with mongodb@5.0.0

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)
mongodb driver version 5 was recently released and it seems that the current version of connect-mongo is not yet compatible with it.

**Comment by TonyDietrich** [comment]
Confirmed

package.json

```
"dependencies": {
    ...
    "connect-mongo": "4.6.0",
   ....
    "mongodb": "5.0.0",
   .....
  }
```

# **npm i**

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: connect-mongo@4.6.0
npm ERR! Found: mongodb@5.0.0
npm ERR! node_modules/mongodb
npm ERR!   mongodb@"5.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer mongodb@"^4.1.0" from connect-mongo@4.6.0
npm ERR! node_modules/connect-mongo
npm ERR!   connect-mongo@"4.6.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: mongodb@4.13.0
npm ERR! node_modules/mongodb
npm ERR!   peer mongodb@"^4.1.0" from connect-mongo@4.6.0
npm ERR!   node_modules/connect-mongo
npm ERR!     connect-mongo@"4.6.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by thuh-reflact** [comment]
Just updating peer dependency will not fix the Problem because of mongo5 driver gots a lot of breaking changes here:

https://github.com/mongodb/node-mongodb-native/releases/tag/v5.0.0

**Comment by MarkoCen** [comment]
Opened a PR to fix this #466

**Comment by mingchuno** [comment]
fixed and released 5.0.0
